releasenotes/notes/oxidize-pmh-ec74e4002510eaad.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade_synthesis:
```

Comment:
We should mark this as `features_synthesis` not as `upgrade_synthesis`; I have just pushed a tiny PR #12715 that fixes it in other places.